### PR TITLE
Add typescript support

### PR DESCRIPTION
**Pull request by jwanner83**

### CODE REVIEW

**Comment by faragos** [comment]
Looks fine to me 👍

File: src/footer.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 var parameters = require('cwd/.webpack/parameters.json')
 var main = require('mainFn')
 
-parameters.forEach((entry) => {
+parameters.forEach(function (entry) {
```
Comment (faragos) [inline src/footer.js:4]: Was this an error in conjunction with the Typescript support?

**Review by faragos — APPROVED** [review]

File: src/footer.js (hunk 1 above)
Comment (jwanner83) [reply src/footer.js:4]: To be honest, I couldn't find the reason why the error suddenly started appearing. I did also try to change the var to const but this also lead to errors. Do you know why this happens?

Comment (faragos) [reply src/footer.js:4]: hmm hard to tell. You used a new Node which supports the arrow function?

Comment (jwanner83) [reply src/footer.js]: I managed to fix it :) It was an eslint problem. After defining that we want to use es6 it works now great 👍

Comment (faragos) [reply src/footer.js]: Thank you a lot :)

Comment (jwanner83) [reply src/footer.js]: No problem :)

**Pull request merged**